noir-projects/aztec-nr/address-note/src/address_note.nr

```diff
@@ -8,18 +8,19 @@ use dep::aztec::{
     },
     oracle::random::random,
     protocol_types::{
-        address::AztecAddress,
-        constants::GENERATOR_INDEX__NOTE_NULLIFIER,
-        hash::poseidon2_hash_with_separator,
-        traits::{Packable, Serialize},
+        address::AztecAddress, constants::GENERATOR_INDEX__NOTE_NULLIFIER,
+        hash::poseidon2_hash_with_separator, traits::Serialize,
     },
 };
 
+// TODO(#12008): Remove the need for the manual import of `Packable` trait here. This is a bug in macros.
```

Comment:
This issue was already present before because we used Packable in the implementations of the pack_content and unpack_content function.s